### PR TITLE
[GO] Added Test Case for aws-sdk-go

### PR DESCRIPTION
**Pull request by windkit**
Create Bucket
List Bucket
Put Object (Small, Single Part Large, Multi Part Large)
Copy Object
Head Object
Get Object (Small, Range Single Part, Range Multi Part)
List Objects
Delete Object

### CODE REVIEW

File: aws-sdk-ruby/leo.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 FileName = "testFile s"
 ChunkSize = 5 * 1024 * 1024  ## 5 MB chunk size
 Bucket = "test" + rand(99999).to_s  ## Dynamic BucketName
-LargeObjSize = 5242880
+LargeObjSize = 52428800
 LargeFilePath = "../temp_data/testFile.large"
```
Comment (mocchira) [inline aws-sdk-ruby/leo.rb]: This issue has already been fixed at develop branch.
So please revert this commit.

**Comment by windkit** [comment]
Reverted

File: aws-sdk-go/leo.go [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                              Region: "us-west-2",
+                              Credentials: cred,
+                              LogLevel: 0})
+
+    bucketStr := aws.String(BucketName)
+
+    _, err := svc.CreateBucket(&s3.CreateBucketInput{Bucket: bucketStr})
+
+    if err != nil {
+        log.Printf("Create Bucket %s Failed, %s\n", BucketName, err)
```
Comment (mocchira) [inline aws-sdk-go/leo.go]: https://golang.org/pkg/log/#Fatalf is more suitable when exiting with logging.
please apply to other places.

File: aws-sdk-go/leo.go [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    log.Printf("Delete Object %s/%s Suceeded\n", BucketName, FileName)
+    log.Println()
+
+    _, err = svc.DeleteBucket(&s3.DeleteBucketInput{Bucket: bucketStr})
+    if err != nil {
+        log.Printf("Delete Bucket %s Failed, %s\n", BucketName, err)
+        return
+    }
+    log.Printf("Delete Bucket %s Suceeded\n", BucketName)
+}
```
Comment (mocchira) [inline aws-sdk-go/leo.go]: Seems missing test cases related to ACL.

Comment (mocchira) [inline aws-sdk-go/leo.go]: As you might notice,
The current aws-client-tests suites written in ONE function due to some reason...
I think we need to refactor code with the manner one function per one test case 
for making it easy to maintain.
Please consider that.

**Comment by windkit** [comment]
Close PR, will include aws-sdk-go together with code refactoring

**Pull request closed without merging**